src/systemcmds/tests/test_dataman.c

```diff
@@ -63,6 +63,16 @@ int test_dataman(int argc, char *argv[]);
 
 #define NUM_MISSIONS_TEST 50
 
+typedef union dataman_max_size_t {
+	struct home_position_s		safe_item;
+	struct mission_item_s		mission_item;
+	struct mission_s			mission;
+	struct fence_vertex_s		vertex;
+	struct dataman_compat_s		compat;
+} dataman_max_size_t;
+
+#define DM_MAX_DATA_SIZE sizeof(dataman_max_size_t)
```

Comment:
Sure but we don't really need the maximum size here. My reasoning is that wenn adding new dataman item types or exchange structs, this place will likely be forgotten to be updated. So by simplifying we make it less prone to mistakes.
Also since we change the internal semantics, there is no global maximum item size anymore, but the size depends on the item type. The tests should take that into account.